lib/vsc/utils/fancylogger.py

```diff
@@ -323,7 +323,9 @@ def raiseException(self, message, exception=None, catch=False):
             exception = self.RAISE_EXCEPTION_CLASS
 
         self.RAISE_EXCEPTION_LOG_METHOD(fullmessage)
-        raise(exception(message)).with_traceback(tb)
+        if tb is not None:
+            raise exception(message).with_traceback(tb)
```

Comment:
why the copy/paste? use a variable and build it up

Reply:
it's a variable less. preference I guess. I'll change.